build/Dockerfile

```diff
@@ -78,3 +78,26 @@ ENV PATH="/opt/phoenix/arm-phoenix/bin:/opt/phoenix/i386-pc-phoenix/bin:/opt/pho
 
 ENTRYPOINT ["./phoenix-rtos-build/build.sh"]
 
+FROM build AS test
+
+# download python3 and qemu
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y --no-install-recommends \
+        python3 \
+        python3-pip \
+        qemu-system-i386 \
+        qemu-system-misc \
+        opensbi \
+        u-boot-qemu \
+        qemu-utils \
+    && rm -rf /var/lib/apt/lists/*
+
+# get python requirements
+RUN pip3 install \
```

Comment:
Here we would like to get requirements.txt from phoenix-rtos-project or phoenix-rtos-tests repository but [PR from phoenix-rtos-test](https://github.com/phoenix-rtos/phoenix-rtos-tests/pull/3) is not merged yet.

Reply:
1. add `--no-cache-dir` so we would not include cached packets in the final image (as per
2. Regarding the requirements.txt - I honestly don't know what the best approach would be:
   - either we would need to remember to rebuild the image each time `phoenix-rtos-test/requirements.txt` would change
   -  or list the requirements here directly and keep them in sync with `requirements.txt`.

As both solutions would result in early failure, I'm ok with both solutions.